examples/hybrid/staggered_nonhydrostatic_model.jl

```diff
@@ -791,155 +798,60 @@ function Wfact_generic!(W, Y, p, dtγ, t)
             norm_sqr(one(ᶜinterp(ᶠw))) *
             ᶜinterp_stencil(one(ᶠw_data))
 
-        # ᶜρₜ = -ᶜdivᵥ(ᶠinterp(ᶜρ) * ᶠw)
-        # ∂(ᶜρₜ)/∂(ᶠw_data) = -ᶜdivᵥ_stencil(ᶠinterp(ᶜρ) * ᶠw_unit)
-        @. ∂ᶜρₜ∂ᶠ𝕄 = -(ᶜdivᵥ_stencil(ᶠinterp(ᶜρ) * one(ᶠw)))
-
-        @. ᶜK = norm_sqr(C123(ᶜuₕ) + C123(ᶜinterp(ᶠw))) / 2
-        thermo_state!(ᶜts, Y, params, ᶜinterp, ᶜK)
-        @. ᶜp = TD.air_pressure(thermo_params, ᶜts)
+        # vertical_transport!(Yₜ.c.ρ, ᶠw, ᶜρ, ᶜρ, dt, Val(:none))
```

Comment:
stale comment / dead code?

Reply:
This is a comment that indicates where the following line comes from.